### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Anandesh-Sharma**
Updating the payload.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,3 @@
 # Payload Library for the Packet Squirrel by Hak5
 
 ![Packet Squirrel](https://www.hak5.org/wp-content/uploads/2017/10/ps2.jpg "Packet Squirrel")
-
```
Comment (sundhaug92) [inline README.md]: No need to remove this

**Comment by sundhaug92** [comment]
I suggest this merge is denied. It only removes information, information that is up to date and whose removal serves now purpose.

**Comment by Anandesh-Sharma** [comment]
Sorry for that....!

**Comment by sundhaug92** [comment]
No need to be, sorry if I was too harsh

**Pull request closed without merging**